packages/valory/skills/safe_deployment_abci/tests/test_behaviours.py

```diff
@@ -89,7 +89,7 @@ def test_validate_behaviour(self) -> None:
         """Run test."""
         self.fast_forward_to_behaviour(
             self.behaviour,
-            self.behaviour_class.behaviour_id,
+            self.behaviour_class.auto_behaviour_id(),
```

Comment:
When dropping, this is a required change. As now the `behaviour_id` at the class level is a property object, not a string.